packages/server/graphql/mutations/helpers/notifications/makeSlackBlocks.ts

```diff
@@ -1,20 +1,40 @@
 import generateUID from '../../../../generateUID'
 
-export const makeSection = (text: string) => ({
-  type: 'section',
-  text: {
-    type: 'mrkdwn',
-    text
+// Slack has the limit of 3000 chars per block
+const SLACK_MAX_BLOCK_CHAR_COUNT = 3000
+const truncationString = `[...]`
```

Comment:
+1 I would have thought `...` is more common for truncation than `[...]` but no strong feelings here

Reply:
@nickoferrall my thought is that `...` could be confused with real `...`

Reply:
Makes sense 👍